### ISSUE TITLE
NTR: Generalized myoclonic seizure

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5184]
"-Generalized-onset motor seizure
-Myoclonic seizure
---Generalized tonic seizure
---Generalized myoclonic-tonic-clonic seizure
---Generalized atonic seizure
---Generalized myoclonic-atonic seizure
---Bilateral tonic-clonic seizure with generalized onset
---Generalized-onset epileptic spasm
---Generalized clonic seizure
---Focal myoclonic seizure
---NTR: Generalized myoclonic seizure
-----Eyelid myoclonia seizure
-----Photosensitive myoclonic seizure
label: Generalized myoclonic seizure
def: A generalized myoclonic seizure is a type of generalized motor seizure characterised by bilateral, sudden, brief (<100 ms) involuntary single or multiple contraction of muscles or muscle groups of variable topography (axial, proximal limb, distal). Myoclonus is less regularly repetitive and less sustained than is clonus.
seeAlso: Generalized epileptic myoclonus, Generalised myoclonic seizure, Generalised epileptic myoclonus
http://webprotege.stanford.edu/RiCnLhdRh6VgSt0KNJ92ib: ILAE 2017
http://purl.org/dc/elements/1.1/source: PMID: 28276064, PMID: 28276060
http://www.w3.org/2000/01/rdf-schema#current_hpo: HP:0002123
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney"
PMID:28276060: https://www.ncbi.nlm.nih.gov/pubmed/PMID:28276060

**Comment by pnrobinson** [comment]
Hello 
!!!

@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney

I will plan on adding terms like this that are synonymous to existing terms in the following way.

1. https://hpo.jax.org/app/browse/term/HP:0002123
The label is the plural form now
I will change it to singular
I will add the plural form as a synoynm that is marked by the label "plural form"
I will update the definition and references as above

-- Is that OK?

For existing terms that do not have an ILAE2017 equivalent, we still need to decide what to do and a short zoom or skype would be good. I think we discuss putting them in a category call 'Pre2017-ILEA' or something like that.

**Comment by DLewisSmith** [comment]
Whilst I am not sure that plural and singular are synonymous (some individuals will only have a particular seizure type once, particularly forms of status epilepticus, which hopefully would only affect most affected individual once in their lives), it seems reasonable to do this for myoclonic seizures as these are frequently recurrent in those affected; I cannot conceive of circumstances in which with an individual would be said to have had exact one myoclonic seizure ever.

However, is it necessary to have the plural as a synonym (what would be the consequences of losing it)?

**Comment by pnrobinson** [comment]
I think that the idea was to keep everything in singular but to record in annotations if a patient has multiple seizures.
